### PR TITLE
Update skeleton

### PR DESCRIPTION
**Pull request by mykola-zomchak**

### CODE REVIEW

File: consts.py [hunk 1]
```diff
@@ -1,2 +1,3 @@
-GAME_URL = 'file:///C:/Program%20Files/WindowsApps/40588MidwayStudios.PAC-MANOriginal_4.0.0.0_x64__svrvwanpjnagm/pacman/index.html'
+GAME_URL = 'file:///C:/Program%20Files/WindowsApps/40588MidwayStudios.PAC-MANOriginal_4.0.0.0_x64__svrvwanpjnagm' \
+           '/pacman/index.html '
 DRIVER_LOCATION = 'resource/chromedriver.exe'
```
Comment (justOleh) [inline consts.py]: Do we need this chromedriver.exe?
Without Selenium it is useless, isn`t it?

File: controller.py [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
-how_long = 10
+how_long = 30
+shooter = ss.shooter()
+parser = PacmanParser()
 bot = CrazyBot()
-with Game() as game:
+
+with Game() as _:
     start = time.time()
     while time.time() - start < how_long:
```
Comment (justOleh) [inline controller.py]: <=

File: screenshooter.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+import numpy
+
+
+class Screenshooter:
+
+    def shot(self) -> numpy.ndarray:
+        return numpy.array(pyautogui.screenshot())
+
+
+def shooter() -> Screenshooter:
```
Comment (justOleh) [inline screenshooter.py]: I don`t understand the advantages of this? Why is this better then use def __init__()?

File: consts.py (hunk 1 above)
Comment (mykola-zomchak) [reply consts.py]: Yeah, we should make a task for moving to python embedded webexecutor.
But it is surely out of the scope of this task

**Pull request merged**